### ISSUE TITLE
Opcode and Operands accessibility

### ISSUE THREAD

**Issue by J4ckKn1ght** [B2R2-org/B2R2#13]
I want to write a custom pattern base on Opcode and Oprands of IntelInstruction but it is a field in internal type. I cannot access it from my library. Can I access that fields? Any problem if I publish InsInfo type?
```F#
[<NoComparison; CustomEquality>]
type InsInfo = internal {
  /// Prefixes.
  Prefixes        : Prefix
  /// REX Prefix.
  REXPrefix       : REXPrefix
  /// VEX information.
  VEXInfo         : VEXInfo option
  /// Opcode.
  Opcode          : Opcode
  /// Operands.
  Operands        : Operands
  /// Instruction size information.
  InsSize         : InsSize
}
```

**Comment by sangkilc** [comment]
Thanks for reporting this bug. I think it should be transparent. Will fix this bug ASAP.

**Comment by sangkilc** [comment]
Fixed in 464375956776606afc35fa9be7232cf58002fcc7.

**Comment by J4ckKn1ght** [comment]
Thanks for your support!!!!
